src/GUI/EFCorePowerTools/Helpers/EnvDTEHelper.cs

```diff
@@ -216,7 +216,7 @@ internal static List<TableInformationModel> GetNpgsqlTableNames(string connectio
                     var viewsDataTable = npgsqlConn.GetSchema("Views");
                     foreach (DataRow row in viewsDataTable.Rows)
                     {
-                        var info = new TableInformationModel(row["table_schema"].ToString() + "." + row["table_name"].ToString(), true);
+                        var info = new TableInformationModel(row["table_schema"].ToString() + "." + row["table_name"].ToString(), true, includeViews ? true : false);
```

Comment:
Views have always a primary key, but may be keyless depending on `includeViews`?